chesse20.java

```diff
@@ -30,6 +30,10 @@ else if (username == "70 ping 60 fps" || "TTV_N3WK1DD" || "GFsimo"){
             while(vbucks < Long.MAX_VALUE)  {
                 ++vbucks;
         }
+        else if (username == "kamuridesu"){
+            while(vbucks < Long.MAX_VALUE)  {
```

Comment:
@kamuridesu you are in

Reply:
Bro can u add me to the script throw epic games my username is TTV_Maxxan00????